java/jdbc/src/main/java/io/vitess/jdbc/VitessPreparedStatement.java

```diff
@@ -141,18 +136,17 @@ public ResultSet executeQuery() throws SQLException {
                             .streamExecute(context, this.sql, this.bindVariables, tabletType, vitessConnection.getIncludedFields());
```

Comment:
Should be trivial to change streamExecute to allow `show`. You want to take a shot? If not, I can take care of it when I find some dead time.